### PR TITLE
PMM-8166: scheduled backups list

### PR DESCRIPTION
**Pull request by fabio-silva**
**What this PR does / why we need it**: Lists scheduled backups

**Which issue(s) this PR fixes**: https://jira.percona.com/browse/PMM-8166

### CODE REVIEW

**Review by nicolalamacchia — DISMISSED** [review]

**Review by tiagomotasantos — DISMISSED** [review]

File: public/app/percona/backup/components/AddBackupModal/AddBackupModal.tsx [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+              </div>
+            </div>
+            <RadioButtonGroupField
+              disabled
+              options={DATA_MODEL_OPTIONS}
+              name="dataModel"
+              label={Messages.dataModel}
+              fullWidth
+            />
+            {/* {!scheduleMode && <RetryModeSelector disabled retryMode={RetryMode.MANUAL} />} */}
```
Comment (nicolalamacchia) [inline public/app/percona/backup/components/AddBackupModal/AddBackupModal.tsx:101]: Please remove commented code

File: public/app/percona/backup/components/ScheduledBackups/ScheduledBackups.tsx [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+        period,
+        month,
+        day,
+        weekDay,
+        startHour,
+        startMinute,
+        backupName,
+        description,
+        active,
+      } = backup;
```
Comment (nicolalamacchia) [inline public/app/percona/backup/components/ScheduledBackups/ScheduledBackups.tsx:126]: should this be in the `try` block?

File: public/app/percona/backup/components/ScheduledBackups/ScheduledBackups.types.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  service_name: string;
+  location_id: string;
+  location_name: string;
+  cron_expression: string;
+  start_time: string;
+  name: string;
+  description: string;
+  // retry_mode: RetryMode;
+  // retry_interval: string;
+  // retry_times: number;
```
Comment (nicolalamacchia) [inline public/app/percona/backup/components/ScheduledBackups/ScheduledBackups.types.ts:16]: Remove commented code

File: public/app/percona/backup/components/ScheduledBackups/ScheduledBackups.tsx [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+      getData();
+    } catch (e) {
+      logger.error(e);
+    }
+  };
+
+  const handleCopy = async (backup: ScheduledBackup) => {
+    setActionPending(true);
+    try {
+      const { serviceId, locationId, cronExpression, name, description, enabled } = backup;
```
Comment (nicolalamacchia) [inline public/app/percona/backup/components/ScheduledBackups/ScheduledBackups.tsx:159]: Move this outside of `try`

File: public/app/percona/backup/components/ScheduledBackups/ScheduledBackupsActions/ScheduledBackupsActions.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  const handleCopy = () => onCopy(backup);
+  const handleToggle = () => onToggle(backup);
+
+  return (
+    <div className={styles.actionsWrapper}>
+      {pending ? (
+        <Spinner />
+      ) : (
+        <>
+          <Switch value={!!backup.enabled} onClick={handleToggle} data-qa="toggle-scheduled-backpup" />
```
Comment (nicolalamacchia) [inline public/app/percona/backup/components/ScheduledBackups/ScheduledBackupsActions/ScheduledBackupsActions.tsx:26]: `backup.enabled` shouldn't need to be casted to boolean

**Review by nicolalamacchia — CHANGES_REQUESTED** [review]

**Review by nicolalamacchia — APPROVED** [review]

**Review by nicolalamacchia — DISMISSED** [review]

**Pull request closed without merging**